<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ChargerImage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>11</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ChargerImage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DetecterContours">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DetecterContours</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="AfficherImageOriginale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="AfficherImageResultat">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="HoughDroite">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HoughDroite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RehaussementContours">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>RehaussementContours</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
